askbot/static/askbot/css/shared.scss

```diff
@@ -43,3 +43,42 @@ abbr[title].timeago {
         background-color: $gray-light;
     }
 }
+
+.tabBar {
+    background: transparent;
+    height: auto;
+
+    .tabsA {
+        background: transparent;
+        border: 1px solid #ccc;
+        border-radius: $border-radius-base;
+        font-size: 12px;
+        height: auto;
+
+        .label {
+            color: $gray-light;
+            line-height: 20px;
+            font-size: 12px;
+            margin: 4px 10px;
+            padding: 0;
+        }
+
+        a {
+            border-left: 1px solid #ccc;
+            height: auto;
+            color: $gray-light;
+            padding: 4px 10px;
+            margin: 0;
+
+            &.on, &[classname="on"] {
```

Comment:
For some reason the `on` property is set as a attribute `classname`, instead of just normally as a class. I wanted to avoid changing the javascript that does that, so I'm just selecting on the attribute.